docs_guidelines/apidocs/tutorials.guidelines.rst

```diff
@@ -0,0 +1,110 @@
+####################
+Tutorials Guidelines
+####################
+
+1. Get the user started
+=======================
+
+1.1 Lower the barrier of entry 
```

Comment:
I think the headings hierarchy looks odd style-wise when it's rendered in the browser. I think maybe the 1. should have `----` underneath and the 1.1, 1.2 etc. should just be bold